src/signTx.c

```diff
@@ -1855,3 +1872,52 @@ void signTx_handleAPDU(
 	VALIDATE(subhandler != NULL, ERR_INVALID_REQUEST_PARAMETERS);
 	subhandler(p2, wireDataBuffer, wireDataSize);
 }
+
+ins_sign_tx_aux_data_context_t* accessAuxDataContext()
```

Comment:
I'm in favor of just `auxDataCtx()`  to keep the name short. (Similarly in other places.)

Reply:
Can be done, sure.

Reply:
Except, it collides with the variable names, and I think the function of the function is better shown in the name as well. It makes checks. Let me think about a better naming for the macros, those are the visible part.